### PR TITLE
AJ-916 Clean up shared CBAS-WDS helm logic for standalone WDS

### PR DESCRIPTION
**Pull request by yuliadub**
Clean up to enable standalone WDS, ticket link [here](https://broadworkbench.atlassian.net/browse/AJ-916).

wds chart now will live here: https://github.com/broadinstitute/terra-helmfile/tree/master/charts/wds

### CODE REVIEW

File: coa-helm/templates/cbas-ui.yaml [hunk 1]
```diff
@@ -27,8 +27,6 @@ data:
     {
       "leoUrlRoot" : "{{ .Values.leonardo.url }}",
       "cbasUrlRoot" : "{{ .Values.relay.path }}/cbas",
-      "cromwellUrlRoot" : "{{ .Values.relay.path }}/cromwell",
-      "wdsUrlRoot" : "{{ .Values.relay.path }}/wds",
-      "wdsInstanceId" : "{{ .Values.persistence.workspaceManager.workspaceId }}"
+      "cromwellUrlRoot" : "{{ .Values.relay.path }}/cromwell"
```
Comment (calypsomatic) [inline coa-helm/templates/cbas-ui.yaml:30]: I think cbas will still need these wds values in order to communicate with it.

File: cromwell-helm/templates/cbas-ui.yaml [hunk 2]
```diff
@@ -22,8 +22,6 @@ data:
   {{ .Values.cbasUI.conf_file }}: |-
     {
       "cbasUrlRoot" : "{{ .Values.ingress.path }}/cbas",
-      "cromwellUrlRoot" : "{{ .Values.ingress.path }}/cromwell",
-      "wdsUrlRoot" : "{{ .Values.ingress.path }}/wds",
-      "wdsInstanceId" : "{{ .Values.persistence.workspaceManager.workspaceId }}"
+      "cromwellUrlRoot" : "{{ .Values.ingress.path }}/cromwell"
```
Comment (calypsomatic) [inline cromwell-helm/templates/cbas-ui.yaml:25]: Ditto here

**Comment by aaronkanzer** [comment]
Other than @calypsomatic comments -- this LGTM

File: coa-helm/templates/cbas-ui.yaml (hunk 1 above)
Comment (yuliadub) [reply coa-helm/templates/cbas-ui.yaml]: I wasnt sure if the .Values.relay.path is still the right url root? after digging around I guess the url hasnt changes, just something else is deploying it, but in this context I guess I am not quite sure if it is safe to assume that Values.relay and Values.ingess are going to have the right path for wds now that the charts are decoupled?

**Review by davidangb — APPROVED** [review]
I think this looks right, definitely want to hear from the Workflow(s) team(s) on this one

**Review by cjllanwarne — APPROVED** [review]

**Pull request merged**